lib/styles/layout/_appswitcher.scss

```diff
@@ -1,7 +1,7 @@
 .neeto-ui-app-switcher__backdrop {
   position: fixed;
   z-index: 99999;
-  left: 80px;
+  left: 72px;
```

Comment:
Reason for this change - https://www.loom.com/share/22a4971a4740476ab05fa15014df61be